recursion/towerofhanoi.js

```diff
@@ -0,0 +1,18 @@
+
+function towerOfHanoiRecur (discCount, source, destination, buffer) {
+    if (discCount >= 1)
+    {
+        towerOfHanoiRecur (discCount-1, source, buffer, destination);
+
+        console.log("Move disc from "+source+" to "+destination);
+
+        towerOfHanoiRecur (discCount-1, buffer, destination, source);
+    }
+
+    return;
+}
+
+towerOfHanoiRecur (3, 'T1', 'T2', 'T3');
```

Comment:
as per tower of hanoi, the number of discs remain the same. Only input here should be the number of discs to start with. Also, it will interesting to return the number of operations it took to shift all discs from a to c.